de_web_plugin.cpp

```diff
@@ -2364,7 +2364,8 @@ void DeRestPluginPrivate::addLightNode(const deCONZ::Node *node)
     auto *device = DEV_GetOrCreateDevice(this, deCONZ::ApsController::instance(), eventEmitter, m_devices, node->address().ext());
     Q_ASSERT(device);
 
-    if (permitJoinFlag)
+    //Profalux device don't have manufactureName and ModelID
+    if ((node->nodeDescriptor().manufacturerCode() != VENDOR_PROFALUX) && permitJoinFlag)
```

Comment:
Can you please change the branching to:

```
if (node->nodeDescriptor().manufacturerCode() == VENDOR_PROFALUX)
{
    //Profalux device don't have manufactureName and ModelID
}
else  if (permitJoinFlag)
{
}
```

Gets a bit easier later on to refactor. We have too many combined branches which I fail to understand after a few weeks/months :)

Reply:
Corrected, but untested this time, but have checked again, it's fine for me